packages/fast-components-react-base/src/hypertext/hypertext.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import Foundation, {HandledProps} from "../foundation";
+import {IHypertextHandledProps, IHypertextManagedClasses, IHypertextUnhandledProps} from "./hypertext.props";
+import {IHypertextClassNameContract, IManagedClasses} from "@microsoft/fast-components-class-name-contracts";
+
+/* tslint:disable-next-line */
+class Hypertext extends Foundation<IHypertextHandledProps & IManagedClasses<IHypertextClassNameContract>,  React.AllHTMLAttributes<HTMLAnchorElement>, {}> {
```

Comment:
Instead of AllHTMLAttributes we'll want HTMLAnchorAttributes

Reply:
Also done? probably